packages/integration-tests/suites/sessions/start-session/subject.js

```diff
@@ -0,0 +1,11 @@
+document.getElementById('start-session').addEventListener('click', () => {
+  Sentry.getCurrentHub().startSession();
```

Comment:
I would prefer if we only tested the `autoSessionTracking` for now, so the sessions that start in pageload/navigation. Let’s not do manual sessions for the time being.